cishouseholds/pipeline/survey_responses_version_2_ETL.py

```diff
@@ -51,9 +58,104 @@ def transform_survey_responses_generic(df: DataFrame) -> DataFrame:
     df = convert_barcode_null_if_zero(df, "blood_sample_barcode")
     df = assign_taken_column(df, "swab_taken", reference_column="swab_sample_barcode")
     df = assign_taken_column(df, "blood_taken", reference_column="blood_sample_barcode")
+    # df = assign_true_if_any(
+    #    df=df,
+    #    column_name_to_assign="sympt_covid_cghfevamn",
+    #    reference_columns=[
+    #        "sympt_covid_cough",
+    #        "sympt_covid_fever",
+    #        "sympt_covid_loss_of_smell",
+    #        "sympt_covid_loss_of_taste",
+    #    ],
+    #    true_false_values=["Yes", "No"],
+    # )
+    # df = assign_true_if_any(
+    #     df=df,
+    #     column_name_to_assign="any_symptoms_last_7_days_or_now",
+    #     reference_columns=["symptoms_last_7_days_any", "think_have_covid_symptoms_now"],
+    #     true_false_values=["Yes", "No"],
+    # )
+    # df = assign_true_if_any(
+    #     df=df,
+    #     column_name_to_assign="symptoms_last_7_days_cghfevamn_symptom_group",
+    #     reference_columns=[
```

Comment:
These are not derived variables, so should be present in the raw input data. Please uncomment to include this one

Reply:
Same for symptom count variables below